src/main/java/winstone/HttpConnectorFactory.java

```diff
@@ -28,31 +24,22 @@
     public boolean start(Map args, Server server) throws IOException {
         // Load resources
         int listenPort = Option.HTTP_PORT.get(args);
-        String listenAddress = Option.HTTP_LISTEN_ADDRESS.get(args);
-        int keepAliveTimeout = Option.HTTP_KEEP_ALIVE_TIMEOUT.get(args);
 
         if (listenPort < 0) {
             return false;
-        } else {
-            ServerConnector connector = createConnector(server, args);
-            connector.setPort(listenPort);
-            connector.setHost(listenAddress);
-            connector.setIdleTimeout(keepAliveTimeout);
-
-            HttpConfiguration config = connector.getConnectionFactory(HttpConnectionFactory.class).getHttpConfiguration();
-            config.addCustomizer(new ForwardedRequestCustomizer());
-            config.setRequestHeaderSize(Option.REQUEST_HEADER_SIZE.get(args));
-
-            server.addConnector(connector);
-            return true;
         }
-    }
+        else {
+            ServerConnectorBuilder scb = new ServerConnectorBuilder()
+                .withServer(server)
+                .withAcceptors(Option.JETTY_ACCEPTORS.get(args))
+                .withSelectors(Option.JETTY_SELECTORS.get(args))
+                .withListenerPort(listenPort)
+                .withListenerAddress(Option.HTTP_LISTEN_ADDRESS.get(args))
+                .withRequestHeaderSize(Option.REQUEST_HEADER_SIZE.get(args))
+                .withKeepAliveTimeout(Option._KEEP_ALIVE_TIMEOUT.get(args));
```

Comment:
#100 FTR

Reply:
@jglick This is what I mean it has been introduced by mistake during this change

Reply:
The _misuse_ of this option was introduced with this change. But the actual option has always (long?) been there. We need to remove the option IMO.